routes/stripe.js

```diff
@@ -73,5 +73,16 @@ module.exports = {
   },
   closeDispute: function(disputeId, callback) {
     stripe.disputes.close(disputeId, callback);
+  },
+  updateCharge: function(chargeId, updateData, callback) {
+    stripe.charges.update(chargeId, updateData, callback);
+  },
+  retrieveCharge: function(chargeId, callback) {
+    stripe.charges.retrieve(chargeId, {
+      expand: ["invoice"]
+    }, callback);
+  },
+  retrieveSubscription: function(customerId, subscriptionId, callback) {
+    stripe.customers.retrieveSubscription(customerId, subscriptionId, callback);
```

Comment:
Ah, I see.

Hm. I say let's just leave it as you have it for right now. Not worth worrying about it right now imo.